cpp/src/hub/solver/riw/riw.hh

```diff
@@ -215,6 +215,13 @@ private:
     typedef Trollout_policy<Domain> RolloutPolicy;
     typedef Texecution_policy ExecutionPolicy;
 
+  private:
+    typedef typename ExecutionPolicy::template atomic<std::size_t>
+        atomic_size_t;
+    typedef typename ExecutionPolicy::template atomic<double> atomic_double;
+    typedef typename ExecutionPolicy::template atomic<bool> atomic_bool;
```

Comment:
These typedefs appear in public methods, why are they private?

Reply:
You're right, I've just committed the change. Thanks!